lib/stbc.gi

```diff
@@ -398,12 +398,20 @@ end);
 InstallGlobalFunction(StabChainBaseStrongGenerators,function(arg)
 local   base,sgs,one,S,  T,  pnt, genlabels;
 
+    if not Length(arg) in [2, 3]
+        or not ForAll([1, 2], i -> IsHomogeneousList(arg[i])) then
```

Comment:
Is it valid to assume that valid bases and strong generating sets will be homogeneous  lists?

Reply:
I think it is but perhaps @hulpke and/or @ChrisJefferson may want to comment?

Reply:
I think it's reasonable